source/module_cell/module_symmetry/symmetry.cpp

```diff
@@ -269,7 +269,16 @@ void Symmetry::analy_sys(const Lattice& lat, const Statistics& st, Atom* atoms,
 
     this->set_atom_map(atoms);
 
-    if (GlobalV::CALCULATION == "relax") this->all_mbl = this->is_all_movable(atoms, st);
+    if (GlobalV::CALCULATION == "relax")
```

Comment:
"cell-relax" also need this autoset.

Reply:
No such problem in cell-relax because the symmetry will be reanalyzed in function `init_after_vc`.